pkg/health/health.go

```diff
@@ -34,9 +35,12 @@ const (
 	Unknown
 )
 
+const Timeout = 5 * time.Minute
```

Comment:
Yes, I was considering that. I added a timeoutSeconds property in LivenessProbe. Hopefully this isn't too invasive.